Documentation/ceph-upgrade.md

```diff
@@ -110,41 +104,50 @@ kubectl -n $ROOK_NAMESPACE get pods
 The Rook toolbox contains the Ceph tools that can give you status details of the cluster with the
 `ceph status` command. Let's look at an output sample and review some of the details:
 ```sh
-> TOOLS_POD=$(kubectl -n $ROOK_NAMESPACE get pod -l "app=rook-ceph-tools" -o jsonpath='{.items[0].metadata.name}')
-> kubectl -n $ROOK_NAMESPACE exec -it $TOOLS_POD -- ceph status
-  cluster:
-    id:     fe7ae378-dc77-46a1-801b-de05286aa78e
-    health: HEALTH_OK
-
-  services:
-    mon: 3 daemons, quorum rook-ceph-mon0,rook-ceph-mon1,rook-ceph-mon2
-    mgr: rook-ceph-mgr0(active)
-    osd: 1 osds: 1 up, 1 in
-
-  data:
-    pools:   1 pools, 100 pgs
-    objects: 0 objects, 0 bytes
-    usage:   2049 MB used, 15466 MB / 17516 MB avail
-    pgs:     100 active+clean
+TOOLS_POD=$(kubectl -n $ROOK_NAMESPACE get pod -l "app=rook-ceph-tools" -o jsonpath='{.items[0].metadata.name}')
+kubectl -n $ROOK_NAMESPACE exec -it $TOOLS_POD -- ceph status
+#  cluster:
```

Comment:
does the `#` help with formatting in some way?

Reply:
It should make it so that if a user copy-pastes it, it won't be executed as though it is a command.

Reply:
For differentiating between commands and output in the docs, a pattern I'm more accustomed to is to add the `$` before the commands, and not have characters before the output, but maybe that's just me.

Reply:
Nearly all of the snippets in the doc are intended to be copy-pasted, and including the `$` in front of a block of commands results in erros like below.
```
If '$' is not a typo you can use command-not-found to lookup the package that contains it, like this:
    cnf $
```

While this may break from typical visual patterns, I believe this is significantly more usable.